graphql/schema.lua

```diff
@@ -85,12 +86,14 @@ function schema:generateTypeMap(node)
 
           local argumentType = argument.__type and argument or argument.kind
           assert(argumentType, 'Must supply type for argument "' .. name .. '" on "' .. fieldName .. '"')
+          argumentType.defaultValue = argument.defaultValue
           self:generateTypeMap(argumentType)
         end
       end
 
       -- HACK: resolve type names to real types
       field.kind = types.resolve(field.kind, self.name)
+      field.defaultValue = defaultValue
```

Comment:
We save this value before the if clause and restore here. What is the idea? The value may be replaced by some other? If so, why the replacement is not correct?

Reply:
Reemoved all unneded lines from schema.lua